css/nav.css

```diff
@@ -1,7 +1,17 @@
 /* --------------------------
   Main Styles
 --------------------------- */
+	.main-header{
+		padding-top: 2em;
+	}
 
+	.site-logo{
+		width: 110px;
+	}
+
```

Comment:
This width should be adequate for filling the screen for small device screen, and also allows room for the right-justified navigation link in the larger screen sizes